### PR TITLE
Use a "title card" instead of video if using Scratch Desktop

### PR DESCRIPTION
**Pull request by paulkaplan**
### Resolves

_What Github issue does this resolve (please include link)?_

- Resolves https://github.com/LLK/scratch-gui/issues/4895

### Proposed Changes

_Describe what this Pull Request does_

Pass in a `showVideos` flag that controls whether to show videos steps or instead show a "title card" consisting of the deck image and name. This is controlled by the `notScratchDesktop()` utility function from the container.

### Test Coverage

_Please show how you have added tests to cover your changes_

Added a `cards` unit test to test whether Video or Image steps are shown based on varying that flag.

Tested the scratchDesktop flag compatibility manually, by using the `isScratchDesktop=true` url param.

### CODE REVIEW

**Comment by paulkaplan** [comment]
@thisandagain just a note, I added the title of the tutorial as well as the image because the cover image doesn't say the name and felt kind of strange without any text. That seem ok?

image

**Comment by paulkaplan** [comment]
@thisandagain I talked with @kosiecki and found two other issues. One, we wanted to shrink the title image (which has a different aspect ratio) to fit so the first card matches the height of the other cards. We did that 
image

We also found that some tutorials _only_ have videos, and brainstormed what to do about that. The most straightforward is to filter those items out of the library for offline mode, because if you loaded them they would contain only the title card, with no steps. I did that, using similar logic to how we filter out the tutorial that requires a specific project to be loaded.

There is still an issue of being able to be linked from a tutorial to one with no steps in the offline editor, which I did not fix, since it felt like it was getting to be a bigger deal than initially expected. I'd like to get confirmation about these workarounds/ this implementation.

File: src/containers/cards.jsx [hunk 1]
```diff
@@ -19,6 +19,7 @@ import {
 
 import CardsComponent from '../components/cards/cards.jsx';
 import {loadImageData} from '../lib/libraries/decks/translate-image.js';
+import {notScratchDesktop} from '../lib/isScratchDesktop';
```
Comment (cwillisf) [inline src/containers/cards.jsx]: Not for this PR, but do you think this should change from indicating Scratch Desktop specifically to some generic `isThis[Not]ABuildWhichShouldSupportRunningWithoutInternetAccess`? If so, any thoughts on a name that's fewer than 60 characters?

Comment (Kenny2github) [reply src/containers/cards.jsx]: `noNeedInternet` maybe? or `shouldHaveInternet` for the reverse?

**Pull request merged**